kinbot/zmatrix.py

```diff
@@ -15,7 +15,6 @@ def make_zmat_from_cart(species, rotor, cart, mode):
     If mode = 1: only those bonds, which generate conformers
     If mode = 2: suply your one rotor in rotor as a list of atom indices
     """
-    #print(species.chemid)
```

Comment:
delete this

Reply:
odd, but this line was not in my code. Let me know I it appears in the next push.